src/pages/Status.jsx

```diff
@@ -64,6 +64,6 @@ class Status extends Component {
     );
   }
 
-};
+}
```

Comment:
This is one of those rules that's linter specific. I'm curious, which linter did you use? I don't have a strong opinion either way, as long as it's consistent. Probably not worth doing in this ticket, but we can add a ticket to the backlog.

Reply:
I had ran this `npx eslint . --ext .js,.jsx,.ts,.tsx --ignore-pattern \"/cypress/*\" --fix` so I believe it is running the config from duos-ui's `.eslintrc`

Reply:
I use the eslint tool in Intelli-j which also enforces this rule. We do have linting rules - I'm surprised that different tools honor those differently